### ISSUE TITLE
Blog post: ** Sentiment Analysis with SOWE in Flux.jl **

### ISSUE THREAD

**Issue by oxinabox** [oxinabox/oxinabox.github.io#3]
Here are notes for a blog post I want to write:

re: https://github.com/FluxML/Flux.jl/issues/251

##  Sentiment Analysis with SOWE in Flux.jl

- Use CorpusLoaders.jl for a sentiment analysis corpus (this needs to be done still)
- Use DataDeps.jl to download word2vec embeddings (and if Word2Vec.jl gets updated to 0.6 use that to load them, maybe, else I have code for it)
- use MLDataUtils to generate a label encoding that maps words to indexes into the pretrained word embeddings
- uses Flux to setup a sum of word embedding -> sentiment  classification
- use MLDataUtils to run evaluation k-fold via cross validation over the corpus,
- with training done in batches of sentences (using MLDataUtils.jl and MultiResolutionIterators.jl) (e

Blocked by:

 - [ ] https://github.com/JuliaText/CorpusLoaders.jl/issues/13
      - [ ] https://github.com/JuliaText/WordTokenizers.jl/issues/3
      - [ ] https://github.com/JuliaText/CorpusLoaders.jl/pull/8
 - [ ] https://github.com/JuliaLang/METADATA.jl/pull/14561

Not actually blocked by:  (But that I think would be nice to have so I can show them working in it)
 -  [ ] https://github.com/JuliaML/MLLabelUtils.jl/pull/17
 - [ ] https://github.com/JuliaML/MLLabelUtils.jl/issues/18
